### PR TITLE
Add click script and PyPI auto-upload via Travis.

### PR DESCRIPTION
**Pull request by doctoryes**
https://openedx.atlassian.net/browse/PLAT-2005

### CODE REVIEW

File: user_util/cli.py [hunk 1]
```diff
@@ -1,18 +1,46 @@
...
+    Console script for user_util to convert usernames/email addresses to retired usernames/email addresses.
+    """
+    try:
+        salt_list = json.loads(salt)
+    except TypeError, ValueError:
+        click.echo("Salt value \"{}\" is invalid JSON.".format(salt))
+        raise
+    results = {}
+    for name in username:
+        # results[name] = get_retired_username(name, salt_list)
```
Comment (macdiesel) [inline user_util/cli.py]: Should this line be removed?

File: user_util/cli.py [hunk 2]
```diff
@@ -1,18 +1,46 @@
...
+        results[name] = user_util.get_retired_username(name, salt_list)
+    for email_addr in email:
+        # results[email_addr] = get_retired_email(email_addr, salt_list)
```
Comment (macdiesel) [inline user_util/cli.py]: Should this line be removed?

**Review by macdiesel — APPROVED** [review]

**Review by bmedx — APPROVED** [review]
Would be cool to see a little more validation on the CLI end of things, but 👍

**Comment by doctoryes** [comment]
OK - I've addressed both your concerns and Travis is now doing its thing.

**Pull request merged**